base_app/admin.py

```diff
@@ -33,7 +33,7 @@ def import_csv(self, request):
         if request.method == "POST":
             uploaded_file = request.FILES['file_upload']
             movies_from_csv = fetch_from_csv(uploaded_file)
-            existing_movie_ids = [movie_obj.imdb_id for movie_obj in Movie.objects.all()]
+            existing_movie_ids = Movie.objects.all().values_list('imdb_id', flat=True)
```

Comment:
While this is ok, we don't need to use .all also. Movie.objects.valus_list is a valid call.